packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -0,0 +1,101 @@
+import asyncStates from '@zooniverse/async-states'
+import React from 'react'
+import PropTypes from 'prop-types'
+import request from 'superagent'
+
+import LightCurveViewer from './LightCurveViewer'
+import locationValidator from '../../helpers/locationValidator'
+
+class LightCurveViewerContainer extends React.Component {
+  constructor () {
+    super()
+    
+    this.state = {
+      loading: asyncStates.initialized,
+      jsonData: null,
+    }
+    
+    //TODO: turn into variables?
+    this.width = 500
```

Comment:
These should probably be in the component, since they determine _how_ the viewer is rendered.

Reply:
Buuuut we need them here because this is where the svg element is being created